.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn lint-staged -p 2
```

Comment:
lint-staged is cool. Just implemented this on another project and really helps to find issues before getting to PR stage